spec/controllers/constants_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+
+def sign_in(user = double('user'))
+  if user.nil?
+    allow(request.env['warden']).to receive(:authenticate!).and_throw(:warden, {:scope => :user})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.